src/html_support_files/odoc.css

```diff
@@ -1070,14 +1070,15 @@ td.def-doc *:first-child {
 
 .odoc-toc li ul {
   margin: 0px;
+  padding-top: 4px;
 }
 
 .odoc-toc ul {
   list-style-type: none;
 }
 
 .odoc-toc ul li {
-  margin: 0;
+  padding: 4px 0;
```

Comment:
Shouldn't this be measured in `em` ? Perhaps `0.2em` as this is half of the removed amount.